src/types.js

```diff
@@ -2,12 +2,17 @@
 import Jss from './Jss'
 import StyleSheet from './StyleSheet'
 import ConditionalRule from './plugins/ConditionalRule'
+import RegularRule from './plugins/RegularRule'
 
 export type ToCssOptions = {
   indent?: number
 }
 
-export type generateClassName = (str: string, rule: Rule) => string
+export type generateClassName = (rule: Rule) => string
+
+// TODO
+// Find a way to declare all types: Object|string|Array<Object>
+export type JssStyle = Object
```

Comment:
Sure, the pobl. is that handleStyle accepts only Style in that example, to be specific to our code base: RegularRule(name, style) where style is always an object. We don't expect arrays there. On the other hand FontFaceRule(selector, style) accepts style as object OR as array in order to support multiple [font faces at once](http://cssinjs.org/json-api?v=v6.5.0#font-face)

So the check `if (Array.isArray)` is local in FontFaceRule and RegularRule, but flow only can detect this if this check would be located in createRule() function which creates all the rules.